Cargo.toml

```diff
@@ -48,6 +48,8 @@ stderrlog = "0.4.1"
 sysconf = ">=0.3.4"
 time = "0.1"
 tiny_http = "0.6"
+rusoto_core = "0.36.0"
+rusoto_dynamodb = "0.36.0"
```

Comment:
Why should `electrs` depend on AWS? I don't use AWS, I don't want this dependency.
But it's even worse, Rusoto says: "Rusoto is in maintenance mode."